### PR TITLE
prep for Async release

### PR DESCRIPTION
**Pull request by JacobBarthelmeh**

### CODE REVIEW

File: wolfcrypt/src/pkcs7.c [hunk 1]
```diff
@@ -8498,8 +8508,21 @@ static int wc_PKCS7_DecryptKtri(PKCS7* pkcs7, byte* in, word32 inSz,
             }
             #endif
             if (ret == 0) {
-                keySz = wc_RsaPrivateDecryptInline(encryptedKey, encryptedKeySz,
-                                                   &outKey, privKey);
+                #ifdef WOLFSSL_ASYNC_CRYPT
+                keySz = 0; /* set initial "ret" value to 0 */
```
Comment (dgarske) [inline wolfcrypt/src/pkcs7.c:8512]: Check indentation here. Please add comment here that this makes the call blocking. Same for pkcs7 change above at 6499.

File: wolfcrypt/test/test.c [hunk 2]
```diff
@@ -21166,8 +21166,6 @@ static int ecc_test_curve_size(WC_RNG* rng, int keySize, int testVerifyCount,
 #endif
 
     ret = wc_ecc_make_key_ex(rng, keySize, userA, curve_id);
-    if (ret != 0)
```
Comment (dgarske) [inline wolfcrypt/test/test.c:21169]: Perhaps moving this to after the async wait? Not sure removing these lines is the right fix. Same for 21193 below.

**Review by dgarske — CHANGES_REQUESTED** [review]

File: wolfcrypt/test/test.c (hunk 2 above)
Comment (JacobBarthelmeh) [reply wolfcrypt/test/test.c:21169]: it looked like there was duplicate checks on ret, one before wc_AsyncWait and one after. The GUI here is jumping over the second check on ret, line 21194 and 21172 in wolfcrypt/test/test.c

Comment (dgarske) [reply wolfcrypt/test/test.c:21169]: Okay my bad, I see it now. GitHub review is not showing the lines with duplicate check below. Good catch.

**Review by dgarske — APPROVED** [review]

**Pull request merged**